components/external_media_links/external_media_list.lua

```diff
@@ -64,7 +64,8 @@ function MediaList._parseArgs(args)
 		types = args.type and Array.map(Array.map(mw.text.split(args.type, ','), String.trim), string.lower) or nil,
 		author = args.author and mw.ext.TeamLiquidIntegration.resolve_redirect(args.author) or nil,
 		subjects = Array.mapIndexes(function(subjectIndex)
```

Comment:
alternative:
```lua
		subjects = Array.map(Array.mapIndexes(function(subjectIndex)
				return args['subject' .. subjectIndex] or args['player' .. subjectIndex]
			end), mw.ext.TeamLiquidIntegration.resolve_redirect)
```
would need testing

Reply:
This alternative works as well (tested via dev).
IMO either way is fine, can we get this merged (soon)?